tests/elasticache.rs

```diff
@@ -2,7 +2,7 @@
 
 extern crate rusoto;
 
-use rusoto::elasticache::{ElastiCacheClient, DescribeCacheClustersMessage};
+use rusoto::elasticache::{Elasticache, ElastiCacheClient, DescribeCacheClustersMessage};
```

Comment:
It's an issue because [CI doesn't run tests that create real AWS resources](https://github.com/rusoto/rusoto/issues/222).  I've been working on getting an AWS account for Rusoto so CI can run them but no luck yet.  There's something new in the works that may get traction for solving it.